src/components/ViewRequest/Actions/DeleteRequest.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogBody,
+  DialogContent,
+  DialogSurface,
+  DialogTitle,
+  DialogTrigger,
+  Tooltip,
+} from "@fluentui/react-components";
+import { DeleteIcon } from "@fluentui/react-icons-mdl2";
+import { useDeleteRequest } from "api/requestsApi";
+import { Navigate, useParams } from "react-router-dom";
+
+const DeleteRequest = () => {
+  const params = useParams();
+  const deleteRequest = useDeleteRequest();
+
+  const deleteHandler = () => {
+    deleteRequest.mutate(Number(params.requestId));
+  };
+
+  return (
+    <>
+      {deleteRequest.isSuccess && <Navigate to="/" />}
```

Comment:
Thoughts on error handling?  Toast notification?  Error icon with details next to button?  Or just leave it like this and they can put in a ticket if it does nothing when they click?